src/scripts.js

```diff
@@ -62,7 +65,7 @@ recipeList.addEventListener("click", function (event) {
 //Functions
 function instantiateClasses(userData, ingredData, recipeData) {
   recipeRepo = new RecipeRepository(ingredData, recipeData);
-  currentUser = new User(userData[0]);
+  currentUser = new User(userData[Math.floor( Math.random() * userData.length )]);
```

Comment:
Good catch on changing this function. This is easy to miss and would have brought our grade down